octoprint_mrbeam/gcodegenerator/img_separator.py

```diff
@@ -326,27 +327,29 @@ def _prepare_img_for_contour_separation(self, monochrome, threshold=255):
         return filtered
 
     def _get_contours(self, img, method=cv2.RETR_EXTERNAL):
+        self.log.info("OpenCV Version:" + cv2.__version__)
         # RETR_EXTERNAL, RETR_LIST, RETR_TREE, RETR_CCOMP
         # see https://docs.opencv.org/ref/master/d9/d8b/tutorial_py_contours_hierarchy.html
         # TODO: switch to RETR_LIST and handle hierarchy recursively
         if isCV2:
+            self.log.info(
+                "OpenCV "
+                + cv2.__version__
+                + " : filtering top level contours with img size cropped by one px on each side"
+            )
             contours, hierarchy = cv2.findContours(
                 img.copy(), method, cv2.CHAIN_APPROX_SIMPLE
             )
+
+        else:
             self.log.info(
                 "OpenCV "
                 + cv2.__version__
                 + " : filtering top level contours with img size cropped by one px on each side"
             )
-
-        else:
-            contours, hierarchy = cv2.findContours(img, method, cv2.CHAIN_APPROX_SIMPLE)
-            if isCV31:
-                self.log.info(
-                    "OpenCV "
-                    + cv2.__version__
-                    + " : filtering top level contours with img size cropped by one px on each side"
-                )
+            _, contours, hierarchy = cv2.findContours(
+                img, method, cv2.CHAIN_APPROX_SIMPLE
+            )
```

Comment:
In version 3.1 (the one used on our devices) it would seem that `findContours` only returns 2 variables.  I would recommend removing the extra mute `_,` because it is not flexible with the number of returned statements as explained in previous comment